src/utils/cache.info.ts

```diff
@@ -194,7 +194,7 @@ export class CacheInfo {
 
   static EsdtProperties(identifier: string): CacheInfo {
     return {
-      key: `esdt:${identifier}`,
+      key: `esdt:v2:${identifier}`,
```

Comment:
I think this is in order to invalidate old cache keys. but we should revert it later

Reply:
This should normally as it is since we have made some incompatible changes and the ttl is one week